python/src/WeibullMaxMuSigma_doc.i.in

```diff
@@ -1,15 +1,24 @@
 %feature("docstring") OT::WeibullMaxMuSigma
 "WeibullMax distribution parameters.
 
+Available constructors:
+    WeibullMaxMuSigma(*mu=1, sigma=1, gamma=0*)
+
```

Comment:
Didn't you remove this section for the previous classes?

Reply:
yes...